opentracing-shim/src/main/java/openconsensus/opentracingshim/SpanShim.java

```diff
@@ -42,20 +43,36 @@ public SpanContext context() {
 
   @Override
   public Span setTag(String key, String value) {
-    span.setAttribute(key, AttributeValue.stringAttributeValue(value));
+    if (key == "span.kind") {
+      // TODO: confirm we can safely ignore span.kind after Span was created
+      // https://github.com/bogdandrutu/openconsensus/issues/42
+    } else if ((key == "error") && value.equalsIgnoreCase("true")) {
```

Comment:
I'm not sure if this is needed or only boolean check is fine

Reply:
A small note: probably we could later add a small method to handle this? To separate the specific semantics, i.e.

```java
boolean handleSpecialTags(Map<String, ?> fields) {
  if ("span.kind".equals(key)) {
    // do specific stuff
    return true;
  }
  if ("error".equals(key)) {
    ...
    return true;
  }

  // No special handling, return, and let setTag() do its normal thing.
  return false;
}
```

Reply:
If it will make code better - it can definitely be done. Keeping it for now